### PR TITLE
cleanup demo, add LICENSE and remove unanalyzed apps

### PR DESCRIPTION
**Pull request by alexhornbake**
https://github.com/ShiftLeftSecurity/product/issues/5453

### CODE REVIEW

**Review by prabhu — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,83 +1,14 @@
...
 
+It is derived from: https://github.com/ShiftLeftSecurity/shiftleft-csharp-example
 
+For more information on how to secure C# applications with ShiftLeft CORE see [the documentation at docs.shiftleft.io](https://docs.shiftleft.io/ngsast/analyzing-applications/c-sharp)
 
+For more information on how to setup ShiftLeft CORE on GitHub Actions see [the documentation at docs.shiftleft.io](https://docs.shiftleft.io/ngsast/workflows/github)
 
+## Applications
 
+| Application Name |Technology, architecture | Packages, API, components | Framework/Version |
```
Comment (hubertp) [inline README.md]: Remove `Packages, API, components` column?

**Review by hubertp — APPROVED** [review]
nit

File: README.md (hunk 1 above)
Comment (alexhornbake) [reply README.md]: > Remove `Packages, API, components` column?

:+1: 

```suggestion
| Application Name |Technology, architecture | Framework/Version |
```

File: README.md [hunk 2]
```diff
@@ -1,83 +1,14 @@
...
+|------------------|-------------------------|-----------------|-------------------|
+| netcoreWebapi | WebAPI, REST | TODO | .NET Core 2.0 |
```
Comment (alexhornbake) [inline README.md]:
```suggestion
| netcoreWebapi | WebAPI, REST | .NET Core 2.0 |
```

**Pull request merged**